art/attacks/evasion/fast_gradient.py

```diff
@@ -334,10 +336,8 @@ def generate(self, x: np.ndarray, y: Optional[np.ndarray] = None, **kwargs) -> n
 
     def _check_params(self) -> None:
 
-        if not (
-            self.norm == "inf"
-            or self.norm >= 1
-        ):
+        norm: float = np.inf if self.norm == "inf" else float(norm)
```

Comment:
Hi @eliegoudout Did you mean to use `self.norm` instead of `norm`? (twice)

Reply:
Yes... 🤦‍♂️

Reply:
Fixed it.
It was even 5 times haha. Locally, PGD and FGM tests pass now for me.

Reply:
:smiley: Thank you!